app/views/patron/_patron.html.erb

```diff
@@ -6,18 +6,30 @@
   <% end %>
   <dt class="col-4 col-md-2 font-weight-normal">Status:</dt>
   <dd class="col-8 col-md-10 patron-status"><%= link_to_if patron.barred?, patron.status, contact_path, class: 'navbar-link', 'data-toggle' =>  'modal', 'data-target' => '#contact-modal-window' %></dd>
+  <% if patron.expired_date && (patron.fee_borrower? || patron.proxy_borrower?) %>
+    <dt class="col-4 col-md-2 font-weight-normal">Privileges expire:</dt>
+    <dd class="col-8 col-md-10 expired-date"><%= l(patron.expired_date, format: :short) %></dd>
+  <% end %>
+  <% if patron.expired_date && patron.expired? %>
+    <dt class="col-4 col-md-2 font-weight-normal">Privileges expired:</dt>
+    <dd class="col-8 col-md-10 expired-date"><%= l(patron.expired_date, format: :short) %></dd>
+  <% end %>
   <% if patron.borrow_limit %>
     <dt class="col-4 col-md-2 font-weight-normal">Borrower limit:</dt>
     <dd class="col-8 col-md-10 patron-status"><%= pluralize(patron.borrow_limit, 'checkout') %></dd>
+  <% elsif patron.expired? %>
```

Comment:
Some of this `if`/`elsif` logic is getting a bit complicated in this view.  I'm thinking about breaking this out into separate partials and having `to_partial_path` return the appropriate thing based on if the patron is expired, or a fee borrower/proxy user (which I think are the two main distinctions we need to make).

Reply:
I'm good with splitting out a special "expired user" partial. It's the condition on L13-16 that really bugs me.

Reply:
I ended up splitting out a fee_borrower partial in addition to the expired user partial to keep things a bit cleaner.  Now there are just simple `if` statements in there (and I threw in tests for all partials for good measure).